feedTheMonster.ts

```diff
@@ -101,7 +100,11 @@ class App {
         element.style.fontFamily = fontFamily;
       }
     });
-   
+  }
+
+  private animateFTMText() {
+    this.titleTextElement.classList.remove("animate");
+    this.titleTextElement.classList.add("animate");
```

Comment:
Why are we removing and then adding the same class?

Reply:
this way we ensure that the title will re-animate when calling this function

Reply:
@bernhardccodev @ashwinnair-chimple pls check